src/tests/components/AutoLogout/AutoLogout.test.tsx

```diff
@@ -0,0 +1,45 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import React from 'react';
+import { act } from 'react-test-renderer';
+
+import AutoLogout from '../../../components/UserAuthentication/AutoLogout';
+
+jest.useFakeTimers();
+
+describe('AutoLogout', () => {
+  afterEach(() => {
+    jest.clearAllTimers();
+  });
+
+  it('should not show warn modal with mouse activity', () => {
```

Comment:
Hi Phil can you add a test case where it will show the the warn modal if there is no activity?